src/Readers/ODataReader.v4/OdcmReader.cs

```diff
@@ -505,11 +500,13 @@ private void WriteMethod(OdcmClass odcmClass, IEdmOperation operation, IEdmOpera
 
             private void WriteProperty(OdcmClass odcmClass, IEdmProperty property)
             {
+                var odcmType = ResolveType(property.Type);
                 var odcmProperty = new OdcmProperty(property.Name)
                 {
                     Class = odcmClass,
                     IsNullable = property.Type.IsNullable,
-                    Type = ResolveType(property.Type),
+                    // get the 'Projection' with default capabilities

```

Comment:
Seems apparent enough from the DefaultOdcmCapabilities. We should remove this comment.